backend/clubs/serializers.py

```diff
@@ -738,6 +738,7 @@ class Meta:
             "approved",
             "code",
             "email",
+            "address",
```

Comment:
I believe that this list is alphabetically sorted, would be nice to keep it this way. Very minor issue, though.